content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -238,55 +219,44 @@ Now you see, we’ve created a new dog with the name Bingo. You can go on to cre
  
 Here is the output:
  
-    Dog running
-    
-    Is the dog running:? true
-    
-    Sleeping
-    
-    is the dog sleeping:? true
-    
-    Dog barking...
-    
-    Is the dog barking:? true
-    
-    Eating
-    
-    Is the dog eating:? true
-    
-    The dog name is: Bingo
-    
-Now that we’ve got this out of the way let's talk about abstraction and encapsulation.
+  Dog running
```

Comment:
Wrap the output as a blockquote or in three backticks (\`\`\`) to more easily distinguish it from the text of the article.